common/scripts/mesh-11s-mr.sh

```diff
@@ -295,39 +295,53 @@ main ()
   echo
   #----------------
 
+  count=0
+
+  rfkill unblock all
+
   if [[ -z "${10}" ]]; then
-    rfkill unblock all
     # multiple wifi options --> can be detected as follows:
     # manufacturer 0x168c = Qualcomm
     # devices = 0x0034 0x003c 9462/988x  11s
     #           0x003e        6174       adhoc
     list="0x0034 0x003c 0x003e"
     for dev in $list; do
-       find_mesh_wifi_device "pci" 0x168c "$dev"
-
-       if [ "$device_list" != "" ]; then
-         for pair in $device_list; do
-           phyname=$(echo "$pair" | cut -f1 -d ',')
-           wifidev=$(echo "$pair" | cut -f2 -d ',')
-
-           echo "  --------------------------------------------------------"
-           echo "- Found: $wifidev $phyname"
-           init_device "$1" "$2" "$3" "$4" "$5" "$6" "$7" "$8" "$9" "$wifidev" "$phyname"
-           if [ "$1" = "ap" ]; then
-             # only one AP is initialised
-             exit 0
-           fi
+      find_mesh_wifi_device "pci" 0x168c "$dev"
+
+      if [ "$device_list" != "" ]; then
+        for pair in $device_list; do
+          phyname=$(echo "$pair" | cut -f1 -d ',')
+          wifidev=$(echo "$pair" | cut -f2 -d ',')
+
+          echo "  --------------------------------------------------------"
+
+          count=$((count+1))
+          echo "- #$count Found: $wifidev $phyname"
+
+          if [ "$count" -gt 1 ]; then
+            # ACS - Automatic Channel Selection, not yet working for wpa/ap
```

Comment:
Is ACS enable not enabled due to missing config parameter in supplicant conf or do u see issue with reported ACS survey data?

Reply:
I had acs=1 in the wpa_supplicant.conf, but this needs to be checked further why it isn't working with 11s.  It would be nice to set second radio to the best channel available.

Reply:
Thanks for the update, we can check if https://patchwork.kernel.org/project/linux-wireless/patch/1552044366-9186-1-git-send-email-pillair@codeaurora.org/ is present and track this via new issue.